### ISSUE TITLE
Question about BEAMnrc parallel computing under Linux.

### ISSUE THREAD

**Issue by CQUswl** [nrc-cnrc/EGSnrc#1123]
When I ran the example BEAM_EX10MeVe under Linux, I noticed a strange phenomenon. When I run it in parallel using BEAMnrc_GUI, the programme does not continue to run after displaying what is shown below.
![image](https://github.com/nrc-cnrc/EGSnrc/assets/114200268/67d98b0c-90fc-4ed5-a072-175f2bf3c08f)
Click "EXECUTE" again to display the same result. At the same time, I found an additional text in the BEAM_EX10MeVe folder with the name &1 and the following contents:
![image](https://github.com/nrc-cnrc/EGSnrc/assets/114200268/fb185458-8c2b-4d00-8af7-099afb3f960e)
However, when I try to run it using the command “egs-parallel”, the program works fine. So I don't know if there is a bug in BEAMnrc_GUI.

**Comment by rtownson** [comment]
Indeed, the GUIs do not support parallel runs unless you modify them to work for your system. They assume you have 'qsub', which is generally not true. In the future we should either remove this feature entirely, or integrate egs-parallel in some way.

For now, just use egs-parallel as you have been.
